cmd/controller-manager/app/controllers.go

```diff
@@ -348,6 +354,37 @@ func addAllControllers(mgr manager.Manager, client k8s.Client, informerFactory i
 		addController(mgr, "volumesnapshot", volumeSnapshotController)
 	}
 
+	// "pvc-autoresizer"
+	if cmOptions.IsControllerEnabled("pvc-autoresizer") {
+		if err := runners.SetupIndexer(mgr, false); err != nil {
+			return err
+		}
+
+		prometheusUrl := "http://" + constants.PrometheusSvcName + "." + constants.KubeSphereMonitoringNamespace + ":" + constants.PrometheusSvcPort
```

Comment:
Shouldn't create prometheusUrl using default service name, the endpoint of Prometheus could be different. Please use the address of  monitoring endpoint, you can reference https://github.com/kubesphere/kubesphere/blob/master/cmd/ks-apiserver/app/options/options.go#L123 for example.